docs/datamap/lucene-datamap-guide.md

```diff
@@ -0,0 +1,213 @@
+# CarbonData Lucene DataMap
+  
+* [Quick Example](#quick-example)
+* [DataMap Management](#datamap-management)
+* [Lucene Datamap](#lucene-datamap-introduction)
+* [Loading Data](#loading-data)
+* [Querying Data](#querying-data)
+* [Data Management](#data-management-with-pre-aggregate-tables)
+
+## Quick example
+1. Download and unzip spark-2.2.0-bin-hadoop2.7.tgz, and export $SPARK_HOME.
+
+2. Package carbon jar, and copy assembly/target/scala-2.11/carbondata_2.11-x.x.x-SNAPSHOT-shade-hadoop2.7.2.jar to $SPARK_HOME/jars.
+   ```shell
+   mvn clean package -DskipTests -Pspark-2.2
+   ```
+
+3. Start spark-shell in new terminal, type :paste, then copy and run the following code.
+   ```scala
+   import java.io.File
+   import org.apache.spark.sql.{CarbonEnv, SparkSession}
+   import org.apache.spark.sql.CarbonSession._
+   import org.apache.spark.sql.streaming.{ProcessingTime, StreamingQuery}
+   import org.apache.carbondata.core.util.path.CarbonStorePath
+   
```

Comment:
Please remove the example code, because pr2268 already provided the executable example.  
Example code should be maintained under examples module, not inside document.

Reply:
ok